### ISSUE TITLE
Unable to parse double column pdf

### ISSUE THREAD

**Issue by smallzhao** [pymupdf/RAG#40]
[0B3168BDCDA63212BC25EDF6681AE1EF.pdf](https://github.com/user-attachments/files/15799424/0B3168BDCDA63212BC25EDF6681AE1EF.pdf)
src_pdf:
<img width="714" alt="image" src="https://github.com/pymupdf/RAG/assets/34087235/8e02800f-aa9d-43de-9789-f7fa3b3cc31f">
dst_md:
<img width="523" alt="image" src="https://github.com/pymupdf/RAG/assets/34087235/bbe9bb68-441a-4b59-b7ec-e5ff416b38bc">

``` 
import pymupdf4llm
md_text = pymupdf4llm.to_markdown("/Users/zhaowenhua/Downloads/期刊/pdf/0B3168BDCDA63212BC25EDF6681AE1EF.pdf", page_chunks=True)

import pathlib
pathlib.Path("/Users/zhaowenhua/Downloads/期刊/pdf/output.md").write_bytes(md_text.encode())

```
I use pymupdf4llm==0.0.5, and I cannot separate the two columns of the PDF. The above is the code and file I used, as well as the generated results. Do I need to configure other parameters to achieve the effect of separating the two columns?

**Comment by JorjMcKie** [comment]
Your PDF is created unusually in that almost every line is a separate text block.
This confuses the column identification algorithm currently.

I have developed a fix which will be published with the next  version.

**Comment by JorjMcKie** [comment]
Partly solved in version 0.0.6.

The solution solves some problems but it is like with table recognition:
There will always be cases that escape a complete detection.
